hardhat.config.ts

```diff
@@ -55,13 +110,80 @@ const config: HardhatUserConfig = {
       },
     },
   },
-  gasReporter: {
-    enabled: process.env.REPORT_GAS !== undefined,
-    currency: "USD",
+  baseURIs: {
```

Comment:
manage base URIs per network here

Reply:
this is really nice! didn't know about this "trick"

Reply:
🙏 , me neither... had to dig for a solution